wallet/wallet.c

```diff
@@ -3087,7 +3170,7 @@ struct amount_msat wallet_total_forward_fees(struct wallet *w)
 	bool res;
 
 	stmt = db_prepare_v2(w->db, SQL("SELECT"
-					" SUM(in_msatoshi - out_msatoshi) "
+					" CAST(SUM(in_msatoshi - out_msatoshi) AS BIGINT)"
```

Comment:
Hard to find exaclty, but I think CAST(NULL) gives 0 as expected.  Might deserve a comment though?

Reply:
The combination of `CAST(COALESCE(field, 0) AS BIGINT)` should take care of this.

Reply:
Just noticed that the `COALESCE` call comes in in a later commit so not touching it here.